### DISCUSSION TITLE
Module for Viral Hepatitis

### DISCUSSION THREAD

**Discussion by caniekwe** [synthetichealth/synthea#1185] [Q&A]
I was wondering if there is some work going on around developing module for viral hepatitis. I will be happy to be pointed in that direction. If there is none, anybody aware of an already existing module close to viral hepatitis I can work out from

**Comment by jawalonoski** [comment]
There is no such module. We don't really have anything close to start from.

**Comment by caniekwe** [comment]
@jawalonoski any suggestions on how to go about this. I am looking at generating data that represents screening and treatment for viral hepatitis.

**Comment by jawalonoski** [comment]
I would suggest starting with looking at: 

- [example module with walk-through](https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-Complete-Example), 
- [module documentation and YouTube video](https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework),
- followed by [our tutorial](https://github.com/synthetichealth/synthea/wiki/Module-Builder-Tutorial).

Once you have the basics down, you can use the [Module Builder](https://synthetichealth.github.io/module-builder/) to build your clinical model -- by editing an existing model or building from scratch.

My suggestion would to think about how you want to model:

- infecting patients with viral hepatitis, 
- diagnosing them, 
- treating them.

**Comment by lhs-open** [comment]
If you write a new module, you may also want to find statistics like infection rate, death rate, dependency on race, age, birth location, vaccination, etc. Viral hepatitis particularly depends on race and immunization status.  You may find viral hepatitis statistics on US CDC website, in US HHS reports or WHO reports.
